### PR TITLE
DGC-138 Added Hero Block, Updated Block Template using Components, Ge…

### PR DESCRIPTION
**Pull request by lkopacz**
…nerated DGC Hero block config.

Note: I didn't include code that Jon already had done in his PR, so this may not work if his code has not been merged in yet. 

- docroot/themes/custom/twentyseventeen/templates/layout/html.html.twig
- docroot/themes/custom/twentyseventeen/templates/layout/page.html.twig

### CODE REVIEW

**Comment by sarahjean** [comment]
@lkopacz Jon's PR is in now, probably want to rebase and retest to make sure this works, then let me know and I will take another look.

**Comment by sarahjean** [comment]
@aweingarten @lkopacz I don't know if you guys talked about this but it might make some sense to use the 'branding block for this and theme that since you'd have like site name, logo, and path variables: https://api.drupal.org/api/drupal/core!modules!system!templates!block--system-branding-block.html.twig/8.2.x

File: config/default/block.block.dgchero.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+uuid: 8a9e58ec-4fd3-4a6a-b92e-0dbea9285023
+langcode: en
+status: true
+dependencies:
+  module:
+    - capitalcamp_blocks
+  theme:
+    - bartik
+id: dgchero
+theme: bartik
```
Comment (aweingarten) [inline config/default/block.block.dgchero.yml]: the theme should not be bartik but the capital camp theme.

**Comment by sarahjean** [comment]
@lkopacz This needs a rebase, probably because I merged Jon's nav PR. It seems like the config isn't placing the block in the correct theme still. As far as testing, should this show on every page, or only on `<front>` once we have it configured?

**Pull request merged**